ui/position.js

```diff
@@ -142,12 +142,13 @@ $.position = {
 	getWithinInfo: function( element ) {
 		var withinElement = $( element || window ),
 			isWindow = $.isWindow( withinElement[ 0 ] ),
-			isDocument = !!withinElement[ 0 ] && withinElement[ 0 ].nodeType === 9;
+			isDocument = !!withinElement[ 0 ] && withinElement[ 0 ].nodeType === 9,
+			hasOffset = element && ( !isWindow && !isDocument );
```

Comment:
Why are these grouped? All the operations are commutative. I'd only expect a group if you were negating the group.